src/sparseml/export/export_data.py

```diff
@@ -196,6 +198,28 @@ def run_inference_with_dict_data(
     return inputs, labels, output
 
 
+def _unnest_past_key_values(output_vals: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Unnest the past key values from the output of the model.
+    (so they exist on the top level of the output dictionary)
+    By default the past key values are nested in a list of unnamed
+    tuples. This function unnests them and names them.
+
+    :param output_vals: The output of the model
+    :return: The output of the model with the past key values unpacked
+    """
+
+    past_key_values = output_vals["past_key_values"]
```

Comment:
nit: can we make this a constant rather than hard coding it? Since its used in a few places

Reply:
This is removed now all together